### ISSUE TITLE
Cursor and history command(zsh) display abnormally

### ISSUE THREAD

**Issue by beyond-fu** [echasnovski/mini.nvim#570]
### Contributing guidelines

- [X] I have read [CONTRIBUTING.md](https://github.com/echasnovski/mini.nvim/blob/main/CONTRIBUTING.md)
- [X] I have read [CODE_OF_CONDUCT.md](https://github.com/echasnovski/mini.nvim/blob/main/CODE_OF_CONDUCT.md)
- [X] I have updated 'mini.nvim' to latest version

### Module(s)

mini.hues

### Description

This plugin is nice.

When I use `habamax` colorscheme, all display well in `toggleterm`(Fig. 1). 
![habamax](https://github.com/echasnovski/mini.nvim/assets/54238106/4210601a-c60a-4667-bb6c-a0d77966838d)
<p align="center"> Fig. 1 habamax</p>

But when I use `mini.hues`, the cursor and the history command(powered by zsh) both can not be displayed in `toggleterm`(Fig. 3).
![mini_hues_noinput](https://github.com/echasnovski/mini.nvim/assets/54238106/3b1ea043-756b-4d4f-8611-370f7b8277de)
<p align="center"> Fig. 2 minu.hues(when nothing typed)</p>

![mini_hues](https://github.com/echasnovski/mini.nvim/assets/54238106/8b74972a-e50a-4d2c-984e-eeac9aba40bc)
<p align="center"> Fig. 3 minu.hues(when just typing ls)</p>

Hope to fix this problem as soon as possible. Sincerely thanks!

### Neovim version

0.9.1

### Steps to reproduce

1. My mini.hues config:
 ```
 {
    "echasnovski/mini.hues",
    version = false,
    config = function()
      require("mini.hues").setup({
        background = "#0E1F25",
        foreground = "#c0c8cc",
        n_hues = 8,
        satuation = "medium",
        accent = "fg",
        plugin = { default = true },
      })
    end,
  },
```
2. My toggleterm config:
 ```
 {
    "akinsho/toggleterm.nvim",
    version = "*",
    keys = {
      { "<C-\\>", "<Cmd>execute v:count . 'ToggleTerm '<CR>", mode = { "i", "n" } },
    },
    config = true,
    opts = {
      size = 20,
      open_mapping = [[<c-\>]],
      hide_numbers = true,
      shade_filetypes = {},
      shade_terminals = true,
      shading_factor = 2,
      start_in_insert = true,
      insert_mappings = true,
      persist_size = true,
      direction = "float",
      close_on_exit = true,
      shell = vim.o.shell,
      float_opts = {
        border = "curved",
        winblend = 0,
        highlights = {
          border = "Normal",
          background = "Normal",
        },
      },
    },
  },
```
3. I'm using `LazyVim`.

### Expected behavior

Just like Fig. 1 above.

### Actual behavior

--------

**Comment by echasnovski** [comment]
Thanks for the issue!

I can not reproduce this, as I don't have the zsh extension showing the history like that.

However, my guess would be that that history uses `vim.g.terminal_color_8` which in 'mini.hues' is the same as background (hence virtually invisible when actually shown). Would you mind try adding the following line **after** the `require('mini.hues').setup()` call:

```lua
vim.g.terminal_color_8 = '#767676'
```

Does this fix the issue?

**Comment by beyond-fu** [comment]
Yes! It works well !!!
```
{
   "echasnovski/mini.hues",
   version = false,
   config = function()
     require("mini.hues").setup({
       background = "#0E1F25",
       foreground = "#c0c8cc",
       n_hues = 8,
       satuation = "medium",
       accent = "fg",
       plugin = { default = true },
     })
     vim.g.terminal_color_8 = '#767676'
   end,
 },
```
Thanks for your quick reply very much! Taking this small problem into consideration for your plugin will be good!

**Comment by echasnovski** [comment]
> Yes! It works well !!!

Great! Thanks for the feedback.

I'll think about it, as it really seems to be an issue.

**Comment by echasnovski** [comment]
Let's keep it open for a while, so that I don't forget about it.

**Comment by echasnovski** [comment]
Yeah, I think having 0-7 and 8-15 terminal colors the same is the best approach here. Otherwise, there is no good way to choose which shade of background/foreground to use to have a universally good user experience. I'll leave it to the user to tweak it to their liking.

Thanks again for raising the issue, though!
